tests/tests/cpld_bmarkfrac_wave_v16_35d

```diff
@@ -55,6 +55,16 @@ export ocn_petlist_bounds="600 839"
 export ice_petlist_bounds="840 887"
 export wav_petlist_bounds="888 1009"
 
+# 12x16 setting
+export TASKS=1700
+export INPES=12
+export JNPES=16
```

Comment:
@DeniseWorthen can you swap INPES and JNPES PE number as 16x12 ?

Reply:
What I have in the _35d tests is identical to ufs-weather mini-release P7a branch. I just made a minor fix to change the indent and comment out the un-used 8x12 settings. 

```
# 12x16 setting
export TASKS=1700
export INPES=16
export JNPES=12
export med_petlist_bounds="0 1151"
export atm_petlist_bounds="0 1239"
export ocn_petlist_bounds="1240 1459"
export ice_petlist_bounds="1460 1539"
export wav_petlist_bounds="1540 1699"
```

Reply:
this looks fine